tests/posix/mt/mt.cpp

```diff
@@ -44,13 +44,12 @@ static PMEMfilepool *global_pfp;
 
 class mt : public pmemfile_test {
 public:
-	long ncpus;
-	pthread_t *threads;
+	unsigned ncpus;
+	std::list<std::thread *> threads;
```

Comment:
I agree with @gbuella, no reason for pointers here. One thing that needs to be mentioned is that std::thread is non-copyable, so remember to either emplace or push r-value references, so they get moved properly.